### ISSUE TITLE
Normalizing new dataset

### ISSUE THREAD

**Issue by AvisP** [benfulcher/hctsa#31]
Hi Ben,

I have been using your HCTSA toolbox for sometime and had a query. First of all thank you for working on the package it has been really useful for my work.

I often face the problem that once I have performed feature extraction on a dataset,normalized the features trained a classifier based on the normalized features and then when I have a new data on which I want to make prediction after feature extraction I do not have the normalization parameters originally used. 

So I have to combine the new and trained data features, normalize together and then separate out the new data. I wish there was something similar to python implementation of fit_transform(). Please let me know if you have any provision in your code for dealing with this.

Best Regards,
Avi

**Comment by benfulcher** [comment]
Hi Avi,
Thanks for the message—glad you're finding it useful for your research.
`TS_normalize` used to have an option to input training indices to learn the normalization on, which was then applied to the remaining (test) data, so that the normalization parameters would not be biased by any knowledge of the test data.
I may find time to implement this, but in the meantime, you could:
1. Make a solution in which the normalization settings are stored and made a new output of TS_normalize
2. After computing features, you could [read in the HCTSA files to python](https://github.com/benfulcher/hctsaAnalysisPython) and then do your classification there.
Let me know your thoughts.
Best,
Ben
